src/main/java/racingcar/Racing.java

```diff
@@ -0,0 +1,20 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Racing {
+    private List<Car> cars;
+
+    public Racing(Integer carCount) {
```

Comment:
```suggestion
        public Racing(List<Car> cars) {
            this.cars = cars
        }
```

`초기화 책임`, 생성자의 책임을 단순하게 유지하는건 어떨까요? 
특별한 로직은 `정적 팩토리 메소드`를 활용해보면 어떨까요?